wolox-training/rof-angular
# Implemented Registration view

## Summary

Implemented Registration view

## Test Plan

* Go to home view.
* Click on register
* Try to create a new user
* Input border colors should change according to validation
* Success and error messages should be shown as you submit the form.

## Trello Card 

https://trello.com/c/rOeX1JCW

## Screenshot

![wbooks - alquila mira y pedi los mejores libros - google chrome_003](https://cloud.githubusercontent.com/assets/2911971/10553843/732e57fe-7434-11e5-8740-4e40762ec35d.png)
